package.json

```diff
@@ -19,7 +19,8 @@
     "postinstall": "is-ci || husky install",
     "partytown": "partytown copylib ./public/~partytown",
     "storybook": "start-storybook --docs -p 6006",
-    "build-storybook": "build-storybook"
+    "build-storybook": "build-storybook",
+    "release": "release-it --ci"
```

Comment:
I think it was to express the current closed beta status.

Reply:
I get it. Let me see if that's it

Reply:
I couldn't achieve what we had before. Release-it creates another "number" for the beta modifier, like `22.30.0-beta.0` and keeps incrementing it (`22.30.0-beta.1`) instead of using the default rule (calendar) and bump it to something like `22.30.1-beta.0`

Reply:
Aaah, okay. Thanks for trying!